services/autoscaling/src/simcore_service_autoscaling/models.py

```diff
@@ -18,6 +18,9 @@ def create_as_empty(cls) -> "Resources":
     def __ge__(self, other: "Resources") -> bool:
         return self.cpus >= other.cpus and self.ram >= other.ram
 
+    def __gt__(self, other: "Resources") -> bool:
```

Comment:
THOUGHT: check if this would make sense here https://docs.python.org/3/library/functools.html#functools.total_ordering. It would define all operations logically based on minimal definitions

Reply:
hmm thanks for the suggestion, but I see few issues here that maybe I do not quite get:
1. my > operator returns True if cpus **OR** ram is bigger
2. my >= operator returns True if cpus **and** ram is bigger or equal
3. I do not need the other operators (at least for now)